app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+    command_name, source_file, destination_file = command_parts
+
+    if len(command_parts) != 3:
+        raise ValueError("Invalid move command format")
+
+    if not os.path.exists(source_file):
+        raise FileNotFoundError(f"Source file not found: {source_file}")
+
+    source_directory = os.path.dirname(source_file)
+    destination_directory = os.path.dirname(destination_file)
+
+    if source_directory == destination_directory:
+        os.rename(source_file, destination_file)
+    else:
+        if not os.path.exists(destination_directory):
+            os.makedirs(destination_directory)
```

Comment:
you can pass `exist_ok=True` so it doesn't trigger exception if the dir already exists

Reply:
Resolved